src/parsers.php

```diff
@@ -475,10 +477,8 @@ function isFallible() : bool
     };
 }
 
-const
-    CONSUME_DO_TRIM = 0x10,
-    CONSUME_NO_TRIM = 0x01
-;
+if(!defined(__NAMESPACE__."\\CONSUME_DO_TRIM")) define(__NAMESPACE__."\\CONSUME_DO_TRIM", 0x10);
+if(!defined(__NAMESPACE__."\\CONSUME_NO_TRIM")) define(__NAMESPACE__."\\CONSUME_NO_TRIM", 0x01);
```

Comment:
No, the stub file exists because IDEs can't recognise define(__NAMESPACE__...). However, we cannot use const here because const is compile-time and therefore cannot be used conditionally.

The stub file has absolutely no effect on the code. There's no point putting this inside the stub variable because it's never intended to be run in the first place. It's just a hack to hint IDEs.